sklearn/utils/testing.py

```diff
@@ -881,6 +881,7 @@ def assert_run_python_script(source_code, timeout=60):
             'cwd': cwd,
             'stderr': STDOUT,
             'env': env,
+            'shell': True
```

Comment:
As far as I know `shell=True` should be avoided. See https://stackoverflow.com/a/3172488/1791279

Why do you think that `cwd` doesn't work without it? I couldn't find anything about it in the docs.

Reply:
Ah it is working correctly. Changing working directly does not automatically add that path to the `PYTHONPATH`. This PR was updated to add cwd to `PYTHONPATH`.